building cluster-autoscaler with clusterapi build tag fails

**Which component are you using?**:

cluster-autoscaler

**What version of the component are you using?**:

Component version: master

**What k8s version are you using (`kubectl version`)?**:

n/a

**What environment is this in?**:

this is during the build process, not while running, i ran the following:
```
$ go build --tags clusterapi -o ./cluster-autoscaler-amd64 
```

**What did you expect to happen?**:

expect that the cluster autoscaler binary is built

**What happened instead?**:

i get an error
```
# k8s.io/autoscaler/cluster-autoscaler/cloudprovider/builder
cloudprovider/builder/cloud_provider_builder.go:46:47: too many arguments in call to buildCloudProvider
        have ("k8s.io/autoscaler/cluster-autoscaler/config".AutoscalingOptions, "k8s.io/autoscaler/cluster-autoscaler/cloudprovider".NodeGroupDiscoveryOptions, *"k8s.io/autoscaler/cluster-autoscaler/cloudprovider".ResourceLimiter, informers.SharedInformerFactory)
        want ("k8s.io/autoscaler/cluster-autoscaler/config".AutoscalingOptions, "k8s.io/autoscaler/cluster-autoscaler/cloudprovider".NodeGroupDiscoveryOptions, *"k8s.io/autoscaler/cluster-autoscaler/cloudprovider".ResourceLimiter)
```

**How to reproduce it (as minimally and precisely as possible)**:
1. check out code
2. `cd cluster-autoscaler`
3. `go build --tags clusterapi -o ./cluster-autoscaler-amd64`

**Anything else we need to know?**:

the fix is relatively simple, update the `buildCloudProvider` function in the `cloudprovider/builder/builder_clusterapi.go` file

--- a/cluster-autoscaler/cloudprovider/builder/builder_clusterapi.go
+++ b/cluster-autoscaler/cloudprovider/builder/builder_clusterapi.go
@@ -23,6 +23,7 @@ import (
 	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider"
 	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider/clusterapi"
 	"k8s.io/autoscaler/cluster-autoscaler/config"
+	"k8s.io/client-go/informers"
 )
 
 // AvailableCloudProviders supported by the cloud provider builder.
@@ -33,7 +34,7 @@ var AvailableCloudProviders = []string{
 // DefaultCloudProvider for machineapi-only build.
 const DefaultCloudProvider = cloudprovider.ClusterAPIProviderName
 
-func buildCloudProvider(opts config.AutoscalingOptions, do cloudprovider.NodeGroupDiscoveryOptions, rl *cloudprovider.ResourceLimiter) cloudprovider.CloudProvider {
+func buildCloudProvider(opts config.AutoscalingOptions, do cloudprovider.NodeGroupDiscoveryOptions, rl *cloudprovider.ResourceLimiter, _ informers.SharedInformerFactory) cloudprovider.CloudProvider {
 	switch opts.CloudProviderName {
 	case cloudprovider.ClusterAPIProviderName:
 		return clusterapi.BuildClusterAPI(opts, do, rl)

